repos/fdbt-site/src/interfaces/index.ts

```diff
@@ -734,3 +734,20 @@ export interface Cap {
     durationAmount: string;
     durationUnits: ExpiryUnit;
 }
+
+export interface MultiTapPricingAttribute {
+    taps: MultiTapPricing[];
+}
+
+export interface MultiTapPricing {
+    tapPrice: string;
```

Comment:
Why do we have tapPriceId in here?

Reply:
Could multiTapPricing instead be an object, like this:

export interface MultiTapPricing {
numberOfTaps: number;
[key: string]: string;
}

so for 3 taps and prices of 2.00, 1.30 and 0.50, the object would look like...

{ 
'1': '2.00',
'2': '1.30',
'3': '0.50'
}

I think this will make it easier to access the values rather than having to map over the array, and we can also get rid of the NUMBER_OF_TAPS_ATTRIBUTE.

We can also use the WithErrors type, so in sessions.ts the line would look like this 

[MULTI_TAPS_PRICING_ATTRIBUTE]: MultiTapPricing | WithErrors<MultiTapPricing>

Reply:
If we do the above, we can remove 3 of the interfaces created here.

Reply:
And we can also use isWithErrors() in our typeguard